src/components/FormProduct/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { addProduct } from 'components/Utils/firestore.js';
+
+export const FormProduct = () => {
+  const inicialStateProduct = {
+    item: '',
+    nextPurchase: 0,
+    LastPurchasedDate: null,
+  };
+
+  const [products, setProduct] = useState(inicialStateProduct);
+
+  const [error, updateError] = useState(false);
```

Comment:
It's a **_little detail_**. According to the official React documentation, we usually use ["setState"](https://es.reactjs.org/docs/hooks-reference.html#usestate) in the hooks, is it possible that you can change it to "`setError`"?. **_Your work works!_** 👍 

Please 🙏, I'll appreciate your change it, but it is your decision to change it or not because it doesn't cause any error 😊.

Reply:
Alejandro thanks a lot for the suggestion. It seems to me that it looks better that way. I already made the changes, you can view them on gitHub.

Reply:
Thanks @macosta08 ! I reviewed the changes and that's ok! 💯